perf/istio-install/values.yaml

```diff
@@ -169,12 +183,12 @@ certmanager:
 
 
 nodeagent:
-  enabled: false
+  enabled: true
   image: node-agent-k8s
   env:
     CA_PROVIDER: "Citadel"
     CA_ADDR: "istio-citadel:8060"
     VALID_TOKEN: true
-    SECRET_GRACE_DURATION: "30m"
+    SECRET_GRACE_DURATION: "10m"
     SECRET_JOB_RUN_INTERVAL: "30s"
-    SECRET_TTL: "1h"
+    SECRET_TTL: "20m"
```

Comment:
Sorry for the late comment. How long does the test run? Can we make it shorter like 5 min?

Reply:
We run these forever pretty much - at least 48hr straight before a release